app.js

```diff
@@ -0,0 +1,60 @@
+
+    if(result === 'win') {
+        wins++;
+        Wins.textContent = wins;
+        Result.textContent = 'You win';
+        resultImage.classList.remove('hidden');
+    } else if(result === 'lose') {
+        losses++;
+        lossDisplay.textContent = losses;
+        Result.textContent = 'You lose...'
+        resultImage.classList.remove('hidden');
+    } else {
+        draws++
+        drawDisplay.textContent = draws
+        Result.textContent = 'Looks like a draw!'
+        resultImage.classList.remove('hidden');
+    }
+});
+
+resetButton.addEventListener ('click', () => {
```

Comment:
This was above and beyond and well done. One thing you might have wanted to add to this event listener would be to hide the result image.